modules/userdirectory/src/main/java/org/opencastproject/userdirectory/InMemoryUserAndRoleProvider.java

```diff
@@ -124,75 +118,90 @@ protected void activate(ComponentContext cc) {
               + "#                                                    #\n"
               + "######################################################");
     }
-
-    // Create the digest user
-    createSystemUsers();
   }
 
   @Override
   public String getName() {
     return PROVIDER_NAME;
   }
 
+  /**
+   * Get a list of all users for the current organization, loading the users from configuration if the organization had
```

Comment:
I don't think this is the list of all users for the current organisation, is it?